dces-drc-integration/src/test/resources/application.yaml

```diff
@@ -0,0 +1,26 @@
+services:
+  maat-api:
+    baseUrl: http://localhost:1111
+    registrationId: MATT_API_REG_ID
+    oAuthEnabled: false
+
+sentry:
+  environment: test
+
+spring:
+  security:
+    oauth2:
+      client:
+        provider:
+          maatapi:
+            token-uri: 34
```

Comment:
These look like test values? Should these be placeholders? Or are they overwritten elsewhere?

Reply:
The test values should really just be in the test yaml though. If you've got things for testing purposes, shelve the changes, so and pull them out when needed. I.e. Placeholders.

Reply:
Wait, why is this in the test section? THere is already an application-test that is being used for testing purposes?